third_party/blink/renderer/modules/webaudio/audio_handler.cc

```diff
@@ -25,6 +25,8 @@ AudioHandler::AudioHandler(NodeType node_type,
     : node_(&node),
       context_(node.context()),
       deferred_task_handler_(&context_->GetDeferredTaskHandler()) {
+  recordreplay::RegisterPointer("AudioHandler", this);
```

Comment:
It would be nice to have a comment describing why this pointer registration is needed.

Reply:
Done.